editor/resources/editor/css/loadscreen.css

```diff
@@ -62,7 +62,7 @@
   font-family: garamond;
   font-size: 48px;
   font-weight: 100;
-  -webkit-font-smoothing: none;
+  /* -webkit-font-smoothing: none; */
```

Comment:
Delete me please

Reply:
Deleted, along with the other lines in that file that were commented out. 🧹